tests/backup/backup_kubevirt_test.go

```diff
@@ -3288,3 +3290,404 @@ var _ = Describe("{CustomBackupRestoreWithKubevirtAndNonKubevirtNS}", Label(Test
+					err = StartAndWaitForVMIMigration(appCtx, context1.TODO())
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verfiying migration of VM [%s] from the node [%s] ", vm.Name, nodeName))
+					wg.Wait()
+					log.Infof("Taking backup of VMs post migration")
+					wg.Add(1)
+					backupName = fmt.Sprintf("%s-%s-%v", "post-migration-backup", scheduledNamespace, time.Now().Unix())
+					go createBackup(backupName, scheduledNamespace)
+					wg.Wait()
+				}
+			}
+		})
+
+		Step("Take backups during rebooting the node in which VM is running", func() {
+			log.InfoD("Take backups during rebooting the node in which VM is running")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, appCtx := range scheduledAppContexts {
+				scheduledNamespace := appCtx.ScheduleOptions.Namespace
+				vms, err := GetAllVMsInNamespace(appCtx.ScheduleOptions.Namespace)
+				log.FailOnError(err, "Failed to get VMs deployed")
+				for _, vm := range vms {
+					createBackup := func(backupName string, namespace string) {
+						defer GinkgoRecover()
+						defer wg.Done()
+						appContextsToBackup := FilterAppContextsByNamespace(scheduledAppContexts, []string{namespace})
+						err = CreateVMBackupWithValidation(ctx, backupName, vms, SourceClusterName, backupLocationName, backupLocationUID, appContextsToBackup,
+							labelSelectors, BackupOrgID, sourceClusterUid, freezeRuleName, freezeRuleUid, unfreezeRuleName, unfreezeRuleUid, true)
+						dash.VerifyFatal(err, nil, fmt.Sprintf("Creation of VM backup and Validation of backup [%s]", backupName))
+						mu.Lock()
+						backupNames = append(backupNames, backupName)
+						backupNamespaceMap[backupName] = namespace
+						defer mu.Unlock()
+					}
+					//Get the node where the vm is scheduled.
+					nodeName, err := GetNodeOfVM(vm)
+					log.FailOnError(err, fmt.Sprintf("Failed to get nodename for the VM %s", vm.Name))
+					nodeObject, err := node.GetNodeByName(nodeName)
+					log.FailOnError(err, fmt.Sprintf("Failed to get node object for the node %s", nodeName))
+					err = RebootNodeAndWaitForPxUp(nodeObject)
+					log.FailOnError(err, "Failed to reboot node and wait till it is up")
+					err = ValidateVMMigration(vm, nodeName)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verfiying VM [%s] is up and running after rebooting the node [%s]", vm.Name, nodeName))
+					log.Infof("Validating all px-backup pods are ready after reboot")
+					err = ValidateAllPodsInPxBackupNamespace()
+					dash.VerifyFatal(err, nil, "px-backup pods verification successful")
+					log.Infof("Taking backup of VMs after node reboot")
+					wg.Add(1)
+					backupName := fmt.Sprintf("%s-%s-%v", "post-reboot-backup", scheduledNamespace, time.Now().Unix())
+					go createBackup(backupName, scheduledNamespace)
+					wg.Wait()
+				}
+			}
+		})
+
+		Step("Upgrade the cluster and verify backups for VM", func() {
```

Comment:
Can we remove this step from here and add it to OCP upgrade test case?
cc: @kshithijiyer-px

Reply:
Discussed with @kshithijiyer-px ,and decided to move to upgrade test. 
created ticket for the same.
https://purestorage.atlassian.net/browse/PB-6694